src/preferences/dialog/dlgprefsounddlg.ui

```diff
@@ -16,16 +16,32 @@
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
+     <item row="0" column="0">
+      <widget class="QLabel" name="apiLabel">
+       <property name="text">
+        <string>Sound API</string>
+       </property>
+       <property name="buddy">
+        <cstring>apiComboBox</cstring>
+       </property>
+      </widget>
+     </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="apiComboBox"/>
      </item>
-     <item row="8" column="0">
-      <widget class="QLabel" name="headDelayLabel">
+     <item row="1" column="0">
+      <widget class="QLabel" name="sampleRateLabel">
        <property name="text">
-        <string>Headphone Delay</string>
+        <string>Sa&amp;mple Rate</string>
```

Comment:
I don't know why, I just copied the text that was there before. We could remove them.